doc/radosgw/multisite.rst

```diff
@@ -48,7 +51,7 @@ mirroring data changes between the Ceph Object Gateway instances and
 this allows for a much simpler configuration scheme and active-active
 configurations. Please note that metadata operations such as creating
 a user would still need to go through the master zone, however data
-operations such as creation of buckets objects can be handled by any
+operations such as creation of buckets or objects can be handled by any
```

Comment:
hmm actually we need to mention that bucket creation, if done on a non-master zone,  internally forwards the request to the master region (this sort of didn't apply to the user creation as we aren't exactly going via the rest api and thus directly create users on a non-master zone)